R/a11y_panel.R

```diff
@@ -0,0 +1,250 @@
+#' Accessibility panel
+#'
+#' @param dashboard_title Title of the host dashboard
+#' @param dashboard_url URL for the host dashboard
+#' @param repo_url URL for the dashboard repository
```

Comment:
Is this optional? Should we make that clear?

Reply:
Done